mods/d2k/rules/aircraft.yaml

```diff
@@ -1,4 +1,4 @@
-carryall.scripted:
+carryall.reinforce:
```

Comment:
See I hadn't considered stances. Having "chase all the way across the map" as the default stance really sucks. We should change the default to Hold position or similar.

Anyway, it was decided not long ago (no idea by who) that we want carryalls untargetable, because they are uncontrollable. I'm fine with it being like this for the moment, until the time comes to do the proper balance. We'll be changing everything there anyway.